content/browser/web_contents/web_contents_impl.cc

```diff
@@ -9173,7 +9171,13 @@ WebContentsImpl* WebContentsImpl::GetFocusedWebContents() {
 }
 
 FrameTree* WebContentsImpl::GetFocusedFrameTree() {
```

Comment:
So I can't seem to get this thing to show the right context, but comments in WebContentsImpl::GetFocusedFrame() suggested we might be doing the wrong thing there w/this change.

Users of GetFocusedFrameWidgetInputHandler look kinda scary, too --- I don't think we want  WebContentsImpl::ExecuteEditCommand to be able to find the outside.

Reply:
I tried and the code quickly becomes hard to understand because of the divergence between secure-embed and guest-view. The current approach lgtm.